packages/flutter/lib/src/material/text_field.dart

```diff
@@ -1149,6 +1151,17 @@ class _TextFieldState extends State<TextField> with RestorationMixin implements
     }
   }
 
+  /// Sets the cursor color to error color from decoration or from theme colorScheme if [_hasError] is true else
+  /// it will return the color from [widget.cursorColor] param if not null. if [widget.cursorColor] is null then it will return default style else return
+  /// color from theme colorScheme.
```

Comment:
Really I think this entire comment is self explanatory by looking at the code and the method signature.  I would just remove it.

Reply:
Removed